media/BUILD.gn

```diff
@@ -113,6 +113,12 @@ rtc_static_library("rtc_media_base") {
     "base/videocommon.h",
     "base/videosourcebase.cc",
     "base/videosourcebase.h",
+
+    #d3d11 frame source files, move these where it makes sense
+    "base/d3d11_frame_buffer.h",
+    "base/d3d11_frame_buffer.cc",
```

Comment:
I would suggest moving the files to `api\video`.

Reply:
Are you sure about this? It would introduce platform-specific code where there seems to be none.

Reply:
Yes, you are right. Nothing platform specific is in `api/video`. I followed implementations for `I420Buffer` and `I010Buffer`. What do you think about moving to `base/win`?

Reply:
There's AndroidVideoBuffer, but all of that is inside `sdk/android/...`. I guess `sdk/windows` would be the "cleanest" solution, but that'd likely require more build script changes, etc. I think I'll just move it back to `media/base` or `media/base/win` for now.

Reply:
Agree about that. It could be moved to `sdk/windows` one day, but it is fine to be left as is.